packages/gallery/src/components/item/itemHelper.js

```diff
@@ -75,3 +75,10 @@ export function shouldCreateVideoPlaceholder(options) {
       options.videoPlay !== GALLERY_CONSTS.videoPlay.AUTO)
   );
 }
+
+export function extractTextItemContent(html) {
+  const span = document.createElement('span');
+  span.innerHTML = html;
+  const content = span.innerText;
+  return content?.trim() || 'Text item';
```

Comment:
@yuvalab at which case there's no content ?

Reply:
i thought in case the content is undefined, but i think there is no case like this. so i can delete.